pomodoro-client.py

```diff
@@ -149,6 +182,9 @@ def get_focused_workspace(i3):
 def create_workspace_policy(disabled_during_pomodoro):
     def allowed_workspace(number):
         pomodoro = get_pomodoro_proxy()
+        if pomodoro is None:
```

Comment:
thanks, I totally forgot about this feature and possibility, as I'm not using it nowadays :smile: